src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -20,12 +21,18 @@
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
 
-    Button signInButton;
-    Label signInLabel;
-    BorderPane signInPane;
-    Scene signInScene, mainScene;
-    Stage thestage;
-    private ClientContext clientContext;
+    private static final double WIDTH = 1000;
+    private static final double HEIGHT = 500;
+
+    private Stage thestage;                         // Holds the scene that user is currently viewing
```

Comment:
nit: Might want to standardize comment formatting between the team, just to make it easier to read.